src/app/Analytics/LocalMonitoring/DatasourceTable.tsx

```diff
@@ -106,7 +95,10 @@ const DatasourceTable = (props: { fetchDatasourcesData }) => {
             ))}
           </Tbody>
         </Table>
-        {isComponentVisible && <ClusterGroupTables clusterGroupData={clusterGroupData} dsname={dsname} />}
+
+        {dataLoaded && dsname.length > 0 &&  <ClusterGroupTables clusterGroupData={dataSourceSelector.dataSourceMetaData}  dsname={dsname}/>}
+      </div>
```

Comment:
remove the closing tag as there is no opening div tag.

Reply:
Ah, thanks! I changed it but forgot to push the changes.